### PR TITLE
Added FixturesLoader

### PR DESCRIPTION
**Pull request by theofidry**
@tshelburne follow up of #242.

### CODE REVIEW

**Comment by theofidry** [comment]
@tshelburne @Seldaek any though on this?

**Comment by sgb-io** [comment]
+1

**Comment by theofidry** [comment]
@Seldaek if you could check this too, would be great :)

File: src/Nelmio/Alice/FixturesLoader.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+     * @param string|array $files filename, glob mask (e.g. *.yml) or array of filenames to load data from, or data
+     *                            array
+     * @param bool         $persist_once
+     *
+     * @return array Persisted objects.
+     */
+    public function load($files, $persist_once = false)
+    {
+        // glob strings to filenames
+        if (!is_array($files)) {
```
Comment (theofidry) [inline src/Nelmio/Alice/FixturesLoader.php]: I don't really like this part: L79-L86. It's about checking if the fixtures files really exists, which is IMO not the responsibility of the loader.

**Comment by Seldaek** [comment]
I'd rather have @tshelburne look over this one because I am not so familiar with the 2.0 bootstrapping code tbh, haven't used alice myself in a while.

**Comment by tshelburne** [comment]
@theofidry Happy to check out all these updates soon - I've been in Iceland for 2 weeks now, heading home in a few days. Will make a point of digging in then.

**Comment by theofidry** [comment]
Cool man, enjoy your holidays :+1: for now, got much to discuss after.

**Comment by tshelburne** [comment]
@theofidry See comment in #242

**Pull request closed without merging**